grpc/optimistic/server.go

```diff
@@ -74,7 +74,13 @@ func (o *AuctionServiceV1Alpha1) GetBidStream(_ *auctionPb.GetBidStreamRequest,
 				bid := auctionPb.Bid{}
 
 				totalCost := big.NewInt(0)
-				effectiveTip := cmath.BigMin(pendingTx.GasTipCap(), new(big.Int).Sub(pendingTx.GasFeeCap(), optimisticBlock.BaseFee))
+				effectiveTip, err := pendingTx.EffectiveGasTip(optimisticBlock.BaseFee)
+				if err != nil {
+					txsTipTooLow.Inc(1)
+					log.Debug("effective tip is too low", "effectiveTip", effectiveTip.String())
+					// don't throw an error but we should avoid streaming this bid
```

Comment:
nit: put the comment above the check block
```suggestion
                                 // don't stream the bid if the tip is below minimum fee
				if err != nil {
					txsTipTooLow.Inc(1)
					log.Debug("effective tip is too low", "effectiveTip", effectiveTip.String())
```